content/includes/survey-tools.html

```diff
@@ -1,6 +1,8 @@
 {% assign auth_required_entrynames = 'claims-status, letters, health-records, messaging, rx, profile-360' | split: ', ' %}
 {% if auth_required_entrynames contains entryname  %}
-  <script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+  {% if buildtype == 'vagovstaging' or buildtype == 'preview' %}
+    <script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+  {% endif %}
```

Comment:
The `content` directory is only for Vets.gov, so I don't think we have to touch this file.